[2022-08-28 23:45:46,309] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: create_and_populate_taxigov_datawarehouse.create_dim_dates_table manual__2022-08-28T23:45:41.667951+00:00 [queued]>
[2022-08-28 23:45:46,315] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: create_and_populate_taxigov_datawarehouse.create_dim_dates_table manual__2022-08-28T23:45:41.667951+00:00 [queued]>
[2022-08-28 23:45:46,316] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-28 23:45:46,316] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-28 23:45:46,316] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-28 23:45:46,325] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): create_dim_dates_table> on 2022-08-28 23:45:41.667951+00:00
[2022-08-28 23:45:46,328] {standard_task_runner.py:52} INFO - Started process 3849 to run task
[2022-08-28 23:45:46,330] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'create_and_populate_taxigov_datawarehouse', 'create_dim_dates_table', 'manual__2022-08-28T23:45:41.667951+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/create_and_populate_taxigov_datawarehouse_dag.py', '--cfg-path', '/tmp/tmp01z27t9u', '--error-file', '/tmp/tmpqx9jldw0']
[2022-08-28 23:45:46,330] {standard_task_runner.py:80} INFO - Job 138: Subtask create_dim_dates_table
[2022-08-28 23:45:46,374] {task_command.py:371} INFO - Running <TaskInstance: create_and_populate_taxigov_datawarehouse.create_dim_dates_table manual__2022-08-28T23:45:41.667951+00:00 [running]> on host ddc8002727e0
[2022-08-28 23:45:46,439] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=create_and_populate_taxigov_datawarehouse
AIRFLOW_CTX_TASK_ID=create_dim_dates_table
AIRFLOW_CTX_EXECUTION_DATE=2022-08-28T23:45:41.667951+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-28T23:45:41.667951+00:00
[2022-08-28 23:45:46,445] {base.py:68} INFO - Using connection ID 'redshift_default' for task execution.
[2022-08-28 23:45:47,777] {dbapi.py:231} INFO - Running statement: 
    DROP TABLE IF EXISTS dim_dates;

    CREATE TABLE dim_dates AS (
        WITH
        distinct_timestamps AS (
            SELECT 
                data_abertura AS ts
            FROM
                raw__taxigov_corridas

            UNION DISTINCT

            SELECT
                data_despacho AS ts
            FROM
                raw__taxigov_corridas 
            
            UNION DISTINCT

            SELECT
                data_local_embarque AS ts
            FROM
                raw__taxigov_corridas 

            UNION DISTINCT

            SELECT
                data_inicio AS ts
            FROM
                raw__taxigov_corridas 

            UNION DISTINCT

            SELECT
                data_final AS ts
            FROM
                raw__taxigov_corridas
        )
        
        SELECT
            DISTINCT
                ts,
                DATE(ts) AS date,
                EXTRACT(MONTH FROM ts) AS month,
                EXTRACT(YEAR FROM ts) AS year,
                EXTRACT(DOW FROM ts) AS day_of_week,
                CASE
                    WHEN EXTRACT(DOW FROM ts) IN (1, 7) THEN TRUE
                    ELSE FALSE 
                END AS is_weekend
        FROM
            distinct_timestamps
    );, parameters: None
[2022-08-28 23:45:50,654] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=create_and_populate_taxigov_datawarehouse, task_id=create_dim_dates_table, execution_date=20220828T234541, start_date=20220828T234546, end_date=20220828T234550
[2022-08-28 23:45:50,674] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-28 23:45:50,701] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
